terraform/modules/shared/dynamo-table/variables.tf

```diff
@@ -14,3 +14,10 @@ variable "ttl_attribute" {
   default     = null
   description = "Enable TTL via a TTL attribute name"
 }
+
+variable "global_secondary_index" {
```

Comment:
With my other comment, we might need to update this to allow passing through a list of maps which can contain a partition and sort key